xpk/README.md

```diff
@@ -106,6 +106,45 @@ cleanup with a `Cluster Delete`.
     --cluster xpk-test
     ```
 
+# How to add docker images to a xpk workload
+
+The default behavior is that the local running directory will be built into
+a docker image by xpk and those files are accessable by `xpk workload create`.
+See below for the two ways that xpk supports configuring docker images for the
+workload. Do not mix arguments from these two ways like `--base-docker-image`
+and `--docker-image` in the same command.
+
+## `--base-docker-image` and `--script-dir`
+Pull in a local directory with scripts or files interested in running in
+`xpk workload create`.
+
+* `--base-docker-image` to set the base image that xpk will start with.
+* `--script-dir` to set which directory to pull into the image.
+* See `python3 xpk/xpk.py workload create --help` for more.
+
+Example with defaults which pulled the local directory into the base image:
+```shell
+echo -e '#!/bin/bash \n echo "Hello world from a test script!"' > test.sh
+python3 xpk/xpk.py workload create --cluster xpk-test \
+--workload xpk-test-workload-base-image --command "bash test.sh" \
+--tpu-type=v5litepod-16 --num-slices=1
+```
+
+## `--docker-image`
+If a user wants to directly set the docker image used, set `--docker-image`
```

Comment:
"If a user wants to directly set the docker image used and not layer in the current working directory"

Reply:
Also probably this example should include a `--base-docker-image` as a command line argument?

Reply:
Since that is the way we recommended.

Reply:
I added --base-docker-image to the example in the `--base-docker-image section`. LMK if that is what you intended.